src/test/java/com/conveyal/gtfs/loader/JDBCTableWriterTest.java

```diff
@@ -1088,10 +1088,10 @@ private static Stream<PatternArguments> createPatternTests() {
         return Stream.of(
             // Add a new stop to the end.
             new PatternArguments(
-                new PatternLocationDTO[] {
+                new PatternLocationDTO[]{
```

Comment:
Revert these changes.

Reply:
Updated my Intellij code style to match this.